plugins/org.jkiss.dbeaver.ext.generic/plugin.xml

```diff
@@ -672,6 +672,24 @@
                     <file type="jar" path="repo:/drivers/dbf/dans-dbf-lib-1.0.0-beta-10.jar" bundle="!drivers.csvjdbc"/>
                     <file type="jar" path="drivers/csvjdbc" bundle="drivers.csvjdbc"/>
                 </driver>
+                <driver
+                    id="jdbcx"
+                    label="JDBCX"
+                    class="io.github.jdbcx.WrappedDriver"
+                    sampleURL="jdbcx:[extension:]vendor://{host}[:{port}][/{database}]"
+                    defaultPort=""
+                    icon="icons/jdbcx_icon.png"
+                    iconBig="icons/jdbcx_icon_big.png"
+                    description="Extended JDBC driver for dynamic multi-language queries with optional bridge server for federated datasource connectivity."
+                    supportedConfigurationTypes="MANUAL,URL"
+                    webURL="https://github.com/jdbcx/jdbcx"
+                    categories="codeql,graphql,prompt,prql,rest,script,shell,sql"
+                    singleConnection="true">
+                    <file type="jar" path="maven:/io.github.jdbcx:jdbcx-driver:RELEASE" load-optional-dependencies="true" bundle="!drivers.jdbcx" />
+                    <file type="license" path="https://www.apache.org/licenses/LICENSE-2.0.txt" bundle="!drivers.jdbcx"/>
+                    <file type="jar" path="drivers/jdbcx" bundle="drivers.jdbcx"/>
+                    <file type="license" path="drivers/apache_lic/LICENSE.txt" bundle="drivers.jdbcx"/>
```

Comment:
I suppose this lines should be removed as no driver and license were added to the drivers repo. We add it only for drivers that we package with DBeaver in PRO versions.